### PR TITLE
DOC: Add data retention and administration policies

### PR DESCRIPTION
**Pull request by effigies**

### CODE REVIEW

**Comment by effigies** [comment]
Suggestions from Adam:

* Bullet list of [NIH's 12 "desirable characteristics"](https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/selecting-a-data-repository#desirable-characteristics-for-all-data-repositories) for a data repository and  how OpenNeuro addresses each of them.
* Note that we are listed in https://www.nlm.nih.gov/NIHbmic/domain_specific_repositories.html

File: docs/policy/data_management_plans.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Data Management Plans
+
+OpenNeuro is a public resource intended to make neuroimaging research data available to as
+broad an audience as possible. This page is intended to help researchers and reviewers evaluate
+whether OpenNeuro meets your criteria for a data host.
+
+For further details, please consult our [](data_retention.md).
```
Comment (ericearl) [inline docs/policy/data_management_plans.md:7]:
```suggestion
For further details, please consult our [Data Retention Policies](data_retention.md).
```

Comment (effigies) [reply docs/policy/data_management_plans.md:7]: This renders with the title of the document.

File: docs/policy/data_management_plans.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+**Provenance**
+: > Has mechanisms in place to record the origin, chain of custody, and any modifications to submitted datasets and metadata.
+
+  All changes to datasets are recorded in the [git][] history of each dataset.
+
+**Retention Policy**
+: > Provides documentation on policies for data retention within the repository.
+
+  These policies may be found at [](data_retention.md).
```
Comment (ericearl) [inline docs/policy/data_management_plans.md:115]:
```suggestion
  These policies may be found at [Data Retention Policies](data_retention.md).
```

**Review by ericearl — COMMENTED** [review]
I know I'm not really a reviewer here typically, but I read these and think they're a great addition!

**Comment by effigies** [comment]
Thanks! Rather than hold this up for more reviewers, I'll let them make suggestions later. Better something than nothing...

**Pull request merged**